packages/textfield/test/textfield.test.ts

```diff
@@ -602,6 +647,49 @@ describe('Textfield', () => {
             : null;
         expect(input).to.not.be.null;
     });
+    it('invalid - boundary-type assertions and title', async () => {
+        const el = await litFixture<Textfield>(
+            html`
+                <sp-textfield
+                    placeholder="Enter your number"
+                    type="url"
+                    value="invalid-email"
+                    invalid="true"
```

Comment:
```suggestion
                    invalid
```
As a boolean attribute, `invalid` doesn't need to be provided a "value".